baseledger/x/bridge/keeper/attestation_handler.go

```diff
@@ -120,7 +121,7 @@ func (a AttestationHandler) Handle(ctx sdk.Context, att types.Attestation, claim
 			return sdkerrors.Wrap(types.ErrIntOverflowAttestation, "invalid supply after UbtDeposit attestation")
 		}
 
-		faucetAddress, err := sdk.AccAddressFromBech32(baseledgercommon.UbtFaucetAddress)
+		faucetAddress, err := sdk.AccAddressFromBech32(a.keeper.GetBaseledgerFaucetAddress(ctx))
```

Comment:
just to confirm, this will always be a valid address because of Validate method below, and we don't need additional checks?

Reply:
Yes, we could remove the additional checks later.